setup.cfg

```diff
@@ -28,3 +28,6 @@ zip_safe = False
 github_owner = readthedocs
 github_repo = readthedocs.org
 
+[codespell]
+ignore-words-list = ba,configurtion,ded,hel,perfom,wile
+skip = *.css,*.fjson,*.js,*.po,*.svg
```

Comment:
We are only running this inside our docs for now, should be in `docs/.codespellrc`

Reply:
Codespell runs successfully across the entire repo.

Reply:
I think it makes sense to run only for `docs/` directory. That way we will reduce the execution time required. It seems the whole project takes +2 minutes to run otherwise, which is a lot: https://github.com/readthedocs/readthedocs.org/pull/8423#issuecomment-902606218

Can you change this to run only on `docs/` directory?

Reply:
Why does it make sense to ignore typos in other parts of the codebase?

Reply:
Oh, I think I missundertood your comment. `tox -e lint` is the one that takes 2+ minutes. It seems that `codespell` only takes 2s to run?

Reply:
codespell is really fast.